### DISCUSSION TITLE
Enlever le cooldown de 30 minutes sur le daily après un switch

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#1852] [Idées acceptées / déjà implémentées]
C'est juste frustrant j'trouve, tant que quelqu'un peut pas faire 3 daily par jour on devrait pouvoir changer d'items sans répercussions, comme c'est sur les autres slots

**Comment by BastLast** [comment]
Ok a la base la mécanique ct que les gens devaient choisir entre avoir un item de combat et avoir un item daily et pas avoir les 2 et pouvoir utiliser les 2. Mais je suis d'accord que la solution actuelle n'est pas terrible. Comment équilibrer le fait qu'avoir les 2 ne soit pas obligatoire ? Peut etre faire en sorte que si tu fais un switch, pendant 48h la puissance de ton item se charge progressivement ? (que ce soit daily ou combat)

**Reply by Pietagorh** [reply]
Perso j'trouve que c'est pas un gros problème qu'on puisse utiliser les deux, ce serait sûrement pas obligatoire à mon avis
C'est un peu comme les potions, tu peux en utiliser de speed/heal et des combats sans problèmes

**Reply by leuhvraiminoryo** [reply]
mais même ce que tu proposes là ajoute rien de positif à l'expérience du joueur, que du négatif. Pourquoi t'veux mettre ce genre de restrictions sur les items ? :pensive:

**Reply by BastLast** [reply]
> mais même ce que tu proposes là ajoute rien de positif à l'expérience du joueur, que du négatif. Pourquoi t'veux mettre ce genre de restrictions sur les items ? :pensive: 

Ballec on met un bonus de 30% qui charge progressivement c'est pareil comme ca ya rien de négatif

**Reply by voltou91** [reply]
Sinon lorsque tu utilise un daily, tu ne peux pas utiliser d'item de combat, et inversement.
En gros tant que tu l'utilises pas ça compte pas comme une utilisation, lorsque tu utilise un objet daily, si tu switch avec ton item de combat, il ne te boostera pas. Et si tu fait un combat dans la journée, alors en switchant sur ton item daily, tu sera bloqué sur l'utilisation du daily.

**Reply by BastLast** [reply]
> Sinon lorsque tu utilise un daily, tu ne peux pas utiliser d'item de combat, et inversement.
> En gros tant que tu l'utilises pas ça compte pas comme une utilisation, lorsque tu utilise un objet daily, si tu switch avec ton item de combat, il ne te boostera pas. Et si tu fait un combat dans la journée, alors en switchant sur ton item daily, tu sera bloqué sur l'utilisation du daily.

Je préfère le bonus

**Reply by Bombafox** [reply]
Ouai mais bon on passe d'un problème de 1h (si on imagine que tu recup ton daily et qu'après tu switch le matin puis le soir)
A un problème ou on doit attendre 2j que ce qu'on veut utiliser soit rentable

**Reply by BastLast** [reply]
> Ouai mais bon on passe d'un problème de 1h (si on imagine que tu recup ton daily et qu'après tu switch le matin puis le soir)
> A un problème ou on doit attendre 2j que ce qu'on veut utiliser soit rentable 

Bah non parce que tu peux l'utiliser comme mtn immédiatement, juste si tu veux un bonus faut pas switch

**Reply by Bombafox** [reply]
Mhh OK

**Comment by BastLast** [comment]
idée acceptée, go #1888 pour la suite
